### PR TITLE
add support for helm 3

### PR DESCRIPTION
**Pull request by jldeen**
Resolves issue #11316 and adds support for Helm 3

Helm 3 removes Tiller and thus removes the "init" command and "--client-only" flag. This is a blocker for those who are prepping to use Helm 3 (currently in RC3 phase).

This PR adds a check for Helm 2. If Helm 2 is specified, `helmTool.arg` is set with `init` and `--client-only` but if helm version specified does not start with "2" then `helmTool.arg` will run `version` instead (best way to check helm 3 usage)

### CODE REVIEW

**Comment by mathieu-benoit** [comment]
FYI: Helm 3.0.0 has just been released officially: https://helm.sh/blog/helm-3-released/ So yep, we need this fix for sure! ;)

**Comment by jldeen** [comment]
@mathieu-benoit I work for Microsoft and I pinged the team internally, too. Hopefully this gets reviewed and merged ASAP.

File: Tasks/HelmInstallerV1/src/helmtoolinstaller.ts [hunk 1]
```diff
@@ -20,10 +19,18 @@ async function configureHelm() {
...
+    // Check if using Helm 2 or Helm 3
+    if (helmVersion.startsWith("2")) {
+        helmTool.arg("init");
+        helmTool.arg("--client-only");
+    } else {
+        helmTool.arg("version");
+    }
+
     return helmTool.exec()
 }
```
Comment (shashankbarsin) [inline Tasks/HelmInstallerV1/src/helmtoolinstaller.ts]: Bump up patch version in task.json and task.loc.json

Comment (jldeen) [reply Tasks/HelmInstallerV1/src/helmtoolinstaller.ts]: Done! Thanks!

**Comment by azooinmyluggage** [comment]
LGTM

**Pull request merged**